system/Validation/Validation.php

```diff
@@ -646,6 +669,9 @@ public function setError(string $field, string $error): ValidationInterface
      */
     protected function getErrorMessage(string $rule, string $field, ?string $label = null, ?string $param = null, ?string $value = null): string
     {
+        $param = $param ?? '';
```

Comment:
how about `(string) $param` ? ref https://3v4l.org/NijdY#v7.3.33

Reply:
@samsonasik What is better?

Reply:
probably just cast it? 

```suggestion
        $param = (string) $param;
```

Reply:
`$param` is null or string. So `$param = $param ?? ''` is no problem.

I'm not sure, but it seems @paulbalandan likes `$param = $param ?? ''`.
Is there any advantage of `(string)`?

Reply:
oh, ok it seems other file have it as well, ok then 👍

Reply:
I don't know of optimization gains but typing `?? ''` is much faster than `(string)` 😂